pkg/admission/quota/clusterresourcequota/admission.go

```diff
@@ -93,12 +93,12 @@ func (q *clusterQuotaAdmission) Validate(ctx context.Context, a admission.Attrib
 	// to its name. Namespaces are cluster level object that shouldn't go into this plugin or it get blocked listing
 	// the namespace that is just being created.
```

Comment:
an alternative approach would be to use the same logic as kubernetes does, which could be a valid approach as well: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/admission.go#L143